Packs/AzureKeyVault/Integrations/AzureKeyVault/AzureKeyVault.yml

```
@@ -0,0 +1,1171 @@
+category: Network Security
+commonfields:
+  id: AzureKeyVault
+  version: -1
+configuration:
+- display: Client ID
+  name: client_id
+  required: true
+  type: 0
+- display: Client Secret (only when using self-deployed Azure application)
```

Let's double-check the grammar of each description or display text please
```suggestion
- display: Client Secret (only when using a self-deployed Azure application)
```

---

Done.